data/filter.go

```diff
@@ -91,3 +100,53 @@ var CorporateInformation = ContentType{
 	Type:            "corporate_information",
 	SubTypes:        []string{"static_foi", "static_page", "static_landing_page", "static_article"},
 }
+
+var errFilterType = errors.New("invalid filter type given")
+
+func setCountZero(categories []Category) []Category {
+	for i, category := range categories {
+		categories[i].Count = 0
```

Comment:
what if `category` is nil? This will panic.

Reply:
categories can't be nil, unless it was updated to be a pointer to []Category, like so:

`func setCountZero(categories []*****Category) []Category {`